### PR TITLE
feat: update homepage apps section

### PR DESCRIPTION
**Pull request by BinaryMuse**
This PR adds a list of companies that use Electron to the top of the apps section on the homepage, and also randomizes the list of apps displayed per request, as discussed and voted on in the website working group meetings.

Fixes https://github.com/electron/electronjs.org/issues/912
Closes https://github.com/electron/electronjs.org/pull/2107

### CODE REVIEW

**Comment by BinaryMuse** [comment]
While working on this, I've discovered that some apps have a white foreground and a transparent background, rendering them essentially invisible on the site. We should track these down and do something about them.

**Comment by BinaryMuse** [comment]
Some more thoughts for app cleanup:

* Remove apps that haven't been updated in a long time (e.g. running on unsupported versions of Electron)
* Fix or remove apps that have empty or broken readmes/images

**Comment by MarshallOfSound** [comment]
> Remove apps that haven't been updated in a long time (e.g. running on unsupported versions of Electron)

I like this, `electron-apps` should automatically flag apps as `outdated: true` or something if the repository URL is set and we can sniff out that their version of Electron is Too Old ™️

**Comment by HashimotoYT** [comment]
# Companies List - Design Mockup

## Option A

The standard list of companies.

Option A

## Option B

Similar to Option A, but with grayscaled logos.

Option B

## Option A+B

Combines A+B, gray logos, on hovering makes colored.

workings

**Comment by BinaryMuse** [comment]
Thanks, @HashimotoYT! I really like all these, but I'm feeling like we might want to lean toward option A as desaturating the logos could be against brand guidelines (in particular, Microsoft mentions only using the full-color logo unless it's not possible to do so).

**Comment by HashimotoYT** [comment]
@BinaryMuse rebase onto `master`, added PoC of companies logos list.

**Comment by BinaryMuse** [comment]
Thanks, @HashimotoYT. Looks good. The Aircall logo feels a little out of place, by itself and without the company name. Is this something they requested? If not, maybe we could use [this image](https://images.ctfassets.net/0kz5nrpxuzuo/Rkm5FQpYdhqBX20PW1RVs/d60c7e60d3b6be9c2da2c22de062cbe8/logo.svg) or something similar.

**Comment by HashimotoYT** [comment]
@BinaryMuse This logotype is taken in a thread in from tracking message in Slack. [Ref](https://electronhq.slack.com/archives/C43E13N0J/p1555431172000600?thread_ts=1553799029.002200&cid=C43E13N0J)

**Comment by BinaryMuse** [comment]
Pushed some changes up to make the images line up at various sizes.:

image

image

image

**Comment by BinaryMuse** [comment]
@electron/wg-website I think this branch is in a good enough spot for a review and a merge. We can always add more companies as time progresses.

File: public/styles/_app.scss [hunk 1]
```diff
@@ -327,3 +327,65 @@
   background: #fff;
   box-shadow: inset 0 0 0 1px #c8e1ff;
 }
+
+.logo-img {
```
Comment (HashimotoYT) [inline public/styles/_app.scss]: Perhaps this doesn't need anymore?

File: views/home.html [hunk 2]
```diff
@@ -214,19 +214,35 @@ <h3 class="f2-light pt-4">{{{localized.need_help.title}}}</h3>
...
       </p>
 
-      <hr>
+      <div class="my-4">
+        <div class="text-center">
+          <ul class="d-flex flex-justify-around flex-wrap list-style-none my-2 my-md-3">
+            {{#each companies}}
+              <li class="featured-company-list-item">
+                <a href="{{ this.url }}" target="_blank">
+                  <img src="{{ site.baseurl }}/images/companies/{{ img }}" alt="{{ this.name }}" class="featured-company-logo">
```
Comment (HashimotoYT) [inline views/home.html]: Please use primer padding, `p-2` (8px)

```suggestion
                  <img src="{{ site.baseurl }}/images/companies/{{ img }}" alt="{{ this.name }}" class="p-2">
```

File: views/home.html [hunk 3]
```diff
@@ -214,19 +214,35 @@ <h3 class="f2-light pt-4">{{{localized.need_help.title}}}</h3>
...
-      <h1 class="f0-light">{{{localized.apps.title}}}</h1>
+      <h1 class="f0-light">{{{localized.companies.title}}}</h1>
       <p class="lead">
-        {{{localized.apps.description}}}
+        {{{localized.companies.description}}}
       </p>
 
-      <hr>
+      <div class="my-4">
+        <div class="text-center">
```
Comment (HashimotoYT) [inline views/home.html]: :fire: `#L216` (this centered anyway 🦄)
```suggestion
```

File: views/home.html [hunk 4]
```diff
@@ -214,19 +214,35 @@ <h3 class="f2-light pt-4">{{{localized.need_help.title}}}</h3>
...
+          <ul class="d-flex flex-justify-around flex-wrap list-style-none my-2 my-md-3">
+            {{#each companies}}
+              <li class="featured-company-list-item">
+                <a href="{{ this.url }}" target="_blank">
```
Comment (HashimotoYT) [inline views/home.html]: [Links to cross-origin destinations are unsafe](https://web.dev/external-anchors-use-rel-noopener/)
```suggestion
                <a href="{{ this.url }}" rel="noopener" target="_blank">
```

File: lib/featured-companies.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+module.exports = [
```
Comment (HashimotoYT) [inline lib/featured-companies.js]: 📝?

Comment (BinaryMuse) [reply lib/featured-companies.js]: Can you please clarify?

**Comment by BinaryMuse** [comment]
> Question about design, why it was changed that way?

I felt the previous design felt visually unbalanced and gave the section a bit of a disorderly look. Some of the logos appeared to be aligned, and others did not.

File: lib/featured-companies.js (hunk 5 above)
Comment (HashimotoYT) [reply lib/featured-companies.js]: e.g., how it's should be updated.

**Pull request merged**